### ISSUE TITLE
Integrate dropzone.js

### ISSUE THREAD

**Issue by andrea-daru** [dustin10/VichUploaderBundle#921]
It's possible to use dropzone or any other upload form type other than the default one?
I can't find documentation about that.
Thank you in advance

**Comment by adrielwerlich** [comment]
Yes vichuploader and dropzone.js would be greate...

**Comment by osavchenko** [comment]
@andrea-daru you can implement your own form field based on vich file (or image) type. Out of the box bundle provide just simple input fields

**Comment by WolfgangDrescher** [comment]
Maybe the recently released Symfony UX allows it integrate this more easily: https://github.com/symfony/ux-dropzone

**Comment by EOL-Fred** [comment]
I'm using VichUploaderBundle, I would like to use Symfony UX Dropzone. In my FormType I replaced VichFileType by DropzoneType, now I have this exception when I try to upload a file :
`Serialization of 'Symfony\Component\HttpFoundation\File\File' is not allowed`
The file is correctly uploaded, the database has been updated, but I can't get rid of this error.

**Comment by tacman** [comment]
Are there any public repos that use this bundle with ux-dropzone?  I'm getting confused on what the entity class and form class should look like.  @WolfgangDrescher , were you able to get this to work?

**Comment by WolfgangDrescher** [comment]
No, I have not yet used ux-dropzone in combination with VichUploaderBundle, sorry.
